src/share/vm/services/threadService.cpp

```diff
@@ -1016,6 +998,27 @@ ThreadsListEnumerator::ThreadsListEnumerator(Thread* cur_thread,
   MutexLockerEx ml(Threads_lock);
 
   for (JavaThread* jt = Threads::first(); jt != NULL; jt = jt->next()) {
+    if (!ThreadsListEnumerator::skipThread(jt, include_jvmti_agent_threads, include_jni_attaching_threads)) {
+      instanceHandle h(cur_thread, (instanceOop) jt->threadObj());
+      _threads_array->append(h);
+    }
+
+    if (EnableCoroutine) {
```

Comment:
do we need to traverse the coroutine list when calling _threads_array->append(h)?

Reply:
In wisp, threads include carrier threads and coroutine converted wisp threads:
We need to get all these so traverse is needed